### PR TITLE
Nutrition: trigger for single word food items

### LINKED ISSUE #3209: Nutrition: Add lists and update triggering
## Problem

The Nutrition Instant Answer doesn't trigger for **single-word food item searches**, such as `avocado`, `beef`, or `steak`:

screen shot 2017-03-24 at 7 28 58 pm

We should trigger on these queries and show an Instant Answer.

## Solution

1. Find one or more lists of popular food items
    - Wikipedia is a good place to start: https://simple.wikipedia.org/wiki/List_of_fruits
2. Update the code to load in each food item as a trigger (see [Currency IA](https://duck.co/ia/view/currency) for an example)
3. Add tests to `t/Nutrition.t` to confirm changes work as expect

## People to notify

@pjhampton

## Get Started
- [ ] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/nutrition

### PR DESCRIPTION
**Pull request by fraed**
## Description of new Instant Answer, or changes

Triggers ia for single worded food items like 'banana' or 'steak' with calories measurement as a default.

## Related Issues and Discussions

Fixes #3209

## People to notify

@moollaza 
@pjhampton 

---

Instant Answer Page: https://duck.co/ia/view/nutrition

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@fraed this is looking great! Deploying to Beta for testing 👍

File: lib/DDG/Spice/Nutrition.pm [hunk 1]
```diff
@@ -24,7 +39,12 @@ handle query_lc => sub {
     if (/^(.+?)\s?$question_regex$attribute_regex(?:\?)?/) {
         return $1 if $1;
     }
-
+    
+    # check for matching fruits, vegetables, animal products, meat, ...
+    if ( grep( /^$_$/, @food_triggers ) ) {
```
Comment (moollaza) [inline lib/DDG/Spice/Nutrition.pm]: @fraed perhaps we should try a naïve approach to handling plurals? If this fails, let's try removing a trailing `s` or replacing `ies` with `y`?

**Comment by moollaza** [comment]
@fraed Thanks! This LGTM :+1:

Excellent work! 💯 🚀

File: share/spice/nutrition/otherfood.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+tofu
+bread
```
Comment (moollaza) [inline share/spice/nutrition/otherfood.txt]: can we add "bagel"?

**Comment by bsstoner** [comment]
This was released to production 🚀

File: lib/DDG/Spice/Nutrition.pm (hunk 1 above)
Comment (fraed) [inline lib/DDG/Spice/Nutrition.pm]: I give it a try during this week. In my opinion we should implement handling plurals on top of this #3262 PR (after review)?

File: share/spice/nutrition/otherfood.txt (hunk 2 above)
Comment (fraed) [inline share/spice/nutrition/otherfood.txt]: Any other food items to add? We should bundle these type of change requests...

**Pull request merged**